PayPalDataCollector/src/main/java/com/braintreepayments/api/PayPalDataCollector.java

```diff
@@ -109,12 +109,12 @@ public void onResult(@Nullable Configuration configuration, @Nullable Exception
                     try {
                         PayPalDataCollectorRequest request = new PayPalDataCollectorRequest()
                                 .setApplicationGuid(getPayPalInstallationGUID(context));
-                        if (clientMetadataId != null) {
-                            request.setClientMetadataId(clientMetadataId);
+                        if (riskCorrelationId != null) {
+                            request.setClientMetadataId(riskCorrelationId);
```

Comment:
Are we able to rename this method to `request.setRiskCorrelationId` as well?

Reply:
Yeah for sure 👍